### PR TITLE
docs: Add field anchors for easier doc navigation

### PR DESCRIPTION
**Pull request by nataliekwong**
## What
Adds Field Anchors for key source docs
- Google Sheets
- Postgres
- Shopify
- File
- Facebook Marketing
- Google Ads

### CODE REVIEW

**Comment by nataliekwong** [comment]
@lmossman would you be able to review this PR? (I'm not sure if anyone else has as much familiarity with Field Anchors)

File: docs/integrations/sources/facebook-marketing.md [hunk 1]
```diff
@@ -77,54 +60,110 @@ To set up Facebook Marketing as a source in Airbyte Open Source, you will first
...
-   10. (Optional) For **Custom Insights Lookback Window**, you may set a window in days to revisit data during syncing to capture updated conversion data from the API. Facebook allows for attribution windows of up to 28 days, during which time a conversion can be attributed to an ad. If you have set a custom attribution window in your Facebook account, please set the same value here. Otherwise, you may leave it at the default value of 28. For more information on action attributions, please refer to [the Meta Help Center](https://www.facebook.com/business/help/458681590974355?id=768381033531365).
+</FieldAnchor>
 
-   :::warning
-   Additional data streams for your Facebook Marketing connector are dynamically generated according to the Custom Insights you specify. If you have an existing Facebook Marketing source and you decide to update or remove some of your Custom Insights, you must also adjust the connections that sync to these streams. Specifically, you should either disable these connections or refresh the source schema associated with them to reflect the changes.
-   :::
```
Comment (lmossman) [inline docs/integrations/sources/facebook-marketing.md:123]: Was it intentional to remove this warning?

File: docs/integrations/sources/google-ads.md [hunk 2]
```diff
@@ -60,26 +60,42 @@ Each Google Ads API developer token is assigned an access level and "permissible
...
+</FieldAnchor>
+
+<FieldAnchor field="start_date">
 8. (Optional) Enter a **Start Date** using the provided datepicker, or by programmatically entering the date in YYYY-MM-DD format. The data added on and after this date will be replicated. (Default start date is 2 years ago)
-9. (Optional) You can use the **Custom GAQL Queries** field to enter a custom query using Google Ads Query Language. Click **Add** and enter your query, as well as the desired name of the table for this data in the destination. Multiple queries can be provided. For more information on formulating these queries, refer to our [guide below](#custom-query-understanding-google-ads-query-language).
+</FieldAnchor>
 
-10. (Required for Manager accounts) If accessing your account through a Google Ads Manager account, you must enter the [**Customer ID**](https://developers.google.com/google-ads/api/docs/concepts/call-structure#cid) of the Manager account.
```
Comment (lmossman) [inline docs/integrations/sources/google-ads.md:76]: Was it intentional to remove this line?

**Review by lmossman — APPROVED** [review]
Called out a couple spots where some text was fully removed as part of the change to add an Anchor. Just wanted to double check that it was intentional.

If those were intentional, then the changes LGTM!

I did not test this locally

File: docs/integrations/sources/facebook-marketing.md (hunk 1 above)
Comment (nataliekwong) [reply docs/integrations/sources/facebook-marketing.md:123]: Ah, this part got moved up in the doc, not removed.  I put it alongside the other info about Custom Insights!

File: docs/integrations/sources/google-ads.md (hunk 2 above)
Comment (nataliekwong) [reply docs/integrations/sources/google-ads.md:76]: I moved this one too :) Up to step 5 where the customer ID instructions are.

**Comment by nataliekwong** [comment]
Closing in favor of https://github.com/airbytehq/airbyte/pull/45885

**Pull request closed without merging**